python/ray/serve/endpoint_state.py

```diff
@@ -64,7 +64,7 @@ def update_endpoint(
 
         existing_route_endpoint = self._get_endpoint_for_route(endpoint_info.route)
         if existing_route_endpoint is not None and existing_route_endpoint != endpoint:
-            logger.warn(
+            logger.debug(
```

Comment:
This isn't ideal since generally users should be warned when they're overwriting existing `route_prefixes`. However, this message always gets logged when a new driver is submitted through the REST API, since it'll replace the old driver's `"/"` prefix. That prints an unnecessary `WARNING` log, even though nothing's wrong.